app/src/main/java/com/babylon/wallet/android/data/gateway/model/TokenPriceResponse.kt

```diff
@@ -0,0 +1,32 @@
+package com.babylon.wallet.android.data.gateway.model
+
+import com.babylon.wallet.android.data.repository.cache.database.TokenPriceEntity
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import rdx.works.core.InstantGenerator
+
+@Serializable
+data class TokenPriceResponse(
+    val id: String,
+    @SerialName(value = "resource_address")
+    val resourceAddress: String,
+    val symbol: String,
+    val name: String,
+    val price: Double,
+    val currency: String
```

Comment:
do we know supported currency values? Maybe this would be better as enum?

Reply:
at the moment everything is USD. Other currencies will follow up in the future.